### DISCUSSION TITLE
Happy for the project to be rototomized

### DISCUSSION THREAD

**Discussion by MarcusNilo** [z-huang/music#1620] [General]
I am very happy to see the project revived.
I kindly ask that you continue to improve this app and add new features, because that way your project will remain as prominent as it always has been.
I saw that another developer recently appropriated the InnerTune project in an immoral and dishonest way.
If possible, I would like to have direct contact with z-huang to share some ideas that could be of good value for the app.

**Comment by z-huang** [comment]
> I saw that another developer recently appropriated the InnerTune project in an immoral and dishonest way.

Can you provide more information?

**Reply by MarcusNilo** [reply]
Yes, I can, but I believe you already know about it.
It's called OpenTune, from the developer Arturo.

https://github.com/Arturo254/InnerTune/releases

**Comment by nvllz** [comment]
> I kindly ask that you continue to improve this app and add new features, because that way your project will remain as prominent as it always has been.

Considering how little the community is involved compared to the number of users, how few people support the project financially, and how much grief there is in the issues tab, it's best not to expect anything more than stability. This year's revival brought numerous useful and missing features, but seeing how it fueled user demands, it may soon become too overwhelming for our dev.
